indexing-service/src/main/java/org/apache/druid/indexing/common/stats/OverlordStatusMonitor.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.indexing.common.stats;
+
+import javax.inject.Inject;
+import org.apache.druid.indexing.overlord.TaskMaster;
+import org.apache.druid.java.util.emitter.service.ServiceEmitter;
+import org.apache.druid.java.util.emitter.service.ServiceMetricEvent;
+import org.apache.druid.java.util.metrics.AbstractMonitor;
+
+/**
+ * Monitor Overlord running status.
+ */
+public class OverlordStatusMonitor extends AbstractMonitor {
+
+  private final TaskMaster taskMaster;
+
+  @Inject
+  public OverlordStatusMonitor(TaskMaster taskMaster) {
+    this.taskMaster = taskMaster;
+  }
+
+  @Override
+  public boolean doMonitor(ServiceEmitter emitter) {
+    final ServiceMetricEvent.Builder builder = new ServiceMetricEvent.Builder();
+
+    builder.setDimension("serviceType", "overlord");
```

Comment:
You can use the service name that is available in the injector like `@Named("serviceName") String serviceName` instead of hardcoding the name of the service in the monitor.

https://github.com/apache/druid/blob/master/server/src/main/java/org/apache/druid/server/DruidNode.java#L114

```suggestion
    builder.setDimension("service", serviceName);
```